code/processes/wdb.q

```diff
@@ -240,21 +241,32 @@ setcompression:{[compression] if[3=count compression;
 				]}
 resetcompression:{setcompression 16 0 0 }
 
+movetohdb:{[dw;hw;pt]
```

Comment:
this function seems unrelated to this PR, should be in a separate PR (I think it maybe is?)